includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-students.php

```diff
@@ -86,6 +89,27 @@ public function get_items( array $filters ): array {
 		return $learners;
 	}
 
+	/**
+	 * Add a user query hook before querying the users.
+	 * This allows for third parties to alter the query.
+	 *
+	 * @since  $$next-version$$
+	 * @access private
+	 *
+	 * @param WP_User_Query $query The user query.
+	 */
+	public function add_pre_user_query_hook( WP_User_Query $query ) {
+		/**
+		 * Fires before the user query is executed.
+		 *
+		 * @hook sensei_reports_overview_students_data_provider_pre_user_query
```

Comment:
Should we add this annotation?
Can't find it in [WP documentation standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters). Maybe it is because of our internal tools or something like that?

Reply:
I remember Donna mentioned it somewhere but I can't find it now. It's related to the [hook docs](https://automattic.github.io/sensei/).